sequential/seq2pat.py

```diff
@@ -151,6 +155,12 @@ def values(self):
         """
         return self._values
 
+    def set_values(self, values):
```

Comment:
Any advices for better practices? This is called in `_mining_batch()` method, which is for internal use then, it might be better to make it to be internal as well.